core/billing_api.php

```diff
@@ -69,10 +70,15 @@ function billing_get_for_project( $p_project_id, $p_from, $p_to, $p_cost_per_hou
 	db_param_push();
 
 	if( ALL_PROJECTS != $p_project_id ) {
-		access_ensure_project_level( config_get( 'view_bug_threshold' ), $p_project_id );
-
-		$t_project_where = ' AND b.project_id = ' . db_param() . ' AND bn.bug_id = b.id ';
-		$t_params[] = $p_project_id;
+		if( $p_subprojects ) {
+			$t_project_ids = user_get_all_accessible_projects(null, $p_project_id);
```

Comment:
While you are here, the implementation of this method checks access level for `VIEWER` rather than `view_bug_threshold` as below.  It would be good if you update this method to match the check in the else block.

Reply:
Should I just replace **VIEWER** for **config_get( 'view_bug_threshold' )** inside **user_get_all_accessible_projects**?
Or should I pass another param to tell it I want **view_bug_threshold** instead of **VIEWER** just in this case?

Reply:
> Should I just replace VIEWER for config_get( 'view_bug_threshold' ) inside user_get_all_accessible_projects?

Yes.